mungegithub/mungers/submit-queue_test.go

```diff
@@ -882,6 +883,21 @@ func TestSubmitQueue(t *testing.T) {
 			isMerged:         true,
 		},
 		{
+			name:            "Fail because vendor-update label is required to be missing but exists",
+			pr:              ValidPR(),
+			issue:           AdditionalLabelIssue("vendor-update"),
```

Comment:
shouldn't this be "vendor-update-approved" ?

Reply:
No, the label is `vendor-update`.